setup.py

```diff
@@ -36,10 +36,10 @@
                         'Markdown>=2.2.0',
                         'PyYAML>=3.10',
                         ],
-      requires=['sh (>=1.0)',
-                'Jinja2 (>=2.6)',
-                'Markdown (>=2.2.0)',
-                'PyYAML (>=3.10)',
+      requires=['sh (>=1.1)',
+                'Jinja2 (>=2.8)',
+                'Markdown (>=2.6.0)',
+                'PyYAML (>=3.12)',
```

Comment:
Nope, I just like to pull things up to the latest version when doing a release as, unless there's a backwards compatibility issue.